dist/index.js

```diff
@@ -288,7 +285,7 @@ async function runCommand(cmd, cmdArgs, env) {
           stdout += buffer.toString();
         },
         stderr(buffer) {
-          stderr += buffer.toString();
+          core.info(buffer.toString());
```

Comment:
This is the same thing `sbom-action` is doing and it is the only reliable way I was able to get the full stderr log. Appending this returned before the process completed and didn't have the full log, so the error at the end is lost :(